src/main/java/org/jboss/pnc/konfluxbuilddriver/Driver.java

```diff
@@ -86,6 +104,19 @@ public BuildResponse create(BuildRequest buildRequest) {
         // TODO: This should be changed to true eventually.
         templateProperties.put("ENABLE_INDY_PROXY", config.indyProxyEnabled());
 
+        try {
+
+            Request notificationCallback = new Request(
+                    Request.Method.PUT,
+                    new URI(StringUtils.appendIfMissing(config.selfBaseUrl(), "/") + "internal/completed"),
+                    Collections.singletonList(new Request.Header(HttpHeaders.CONTENT_TYPE_STRING, MediaType.APPLICATION_JSON)),
+                    buildRequest.completionCallback());
+
+            templateProperties.put("NOTIFICATION_CONTEXT", new ObjectMapper().writeValueAsString(notificationCallback));
```

Comment:
Injet ObjectMapper, that we can use the same instance (with potential custom config) everywhere.